MAUI/DataForm/CustomDataFormItem.md

```diff
@@ -0,0 +1,176 @@
+                "United Kingdom",
+                "France",
+                "Belgium",
+                "Germany"
+            };
+        }
+
+        public DataFormModel Details
+        {
+            get { return this.details; }
+            set { this.details = value; }
+        }
+        public IList<string> CountryNames
+        {
+            get { return this.countryNames; }
+            set { this.countryNames = value; }
+        }
+    }
+
+    private IList GetItemSource(string sourceName) 
+    { 
+        var list = new List<string>(); 
+        if (sourceName == "Countries") 
+        { 
+            list.Add("India");
+            list.Add("Japan");
+            list.Add("China");
+            list.Add("USA");
+            list.Add("America");
+            list.Add("UK");
+        } 
+        return list; 
+    }
+{% endhighlight %}
+{% endtabs %}
+
+### Dynamically add custom dataform items
+Support has been provided to dynamically add the dataform items to collections using the [Items]`Link` property of `SfDataForm`.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.Items.Add(new DataFormComboBoxItem() 
+				    { FieldName = "StateName",  
+					    ItemsSource = this.GetItemSource("StateName"), 
+					    PlaceHolderText = "Select a State" 
+				    });
+				
+    dataForm.Items.Add(new DataFormSwitchItem() 
+				    { FieldName = "Save"  });
+{% endhighlight %}
+{% endtabs %}
+
+`AddCustomItemsimages`
+
+### Dynamically remove custom dataform items
+Support has been provided to dynamically remove the dataform items from collections using the [Items]`Link` property of `SfDataForm`.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.Items.RemoveAt(2);
+
+{% endhighlight %}
+{% endtabs %}
+
+`images/RemoveCustomItems`
+
+### Dynamically clear custom dataform items
+Support has been provided to dynamically clear the dataform items using the [Items] property of `SfDataForm`.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.Items.Clear();
+
+{% endhighlight %}
+{% endtabs %}
+
+`images/ClearCustomItems`
+
+### Dynamically reset custom dataform items
+Support has been provided to reset the dataform items using the [Items] property of `SfDataForm` and we can reset two or more dataform items at a time.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    var item = dataForm.Items[2];
+    item = new DataFormTextItem() { FieldName = "Age" };
+    dataForm.Items[2] = item;
+
+    var item = dataForm.Items[3];
+    item = new DataFormDateItem() { FieldName ="Date Of Birth", RowOrder = 2, ItemsOrderInRow = 1, ColumnSpan = 3 });
+    
+{% endhighlight %}
+{% endtabs %}
+
+
+### Dynamically add custom dataform group items
+Support has been provided to dynamically add custom group items using [Items] property of `SfDataForm`.
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    DataFormGroupItem dataFormGroupItem = new DataFormGroupItem();
+    dataFormGroupItem.GroupName = "GroupItem";
```

Comment:
change GroupName to Name from GroupItem in all applicable places

Reply:
groupName to name changed